### PR TITLE
[MRS] Refactor

### LINKED ISSUE #430: [MRS] API Updated
https://confluence.tsi-dev.otc-service.com/display/DR/RM-4228-MRS-API

### PR DESCRIPTION
**Pull request by Aloento**

### CODE REVIEW

**Review by anton-sidelnikov — DISMISSED** [review]

File: openstack/mrs/v1/job/Create.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+type CreateDistCpOpts struct {
+	// 5: DistCp, importing and exporting data.
+	JobType int `json:"job_type"`
+	// Job name
+	// Contains only 1 to 64 letters, digits, hyphens (-), and underscores (_).
+	// NOTE:
+	// Identical job names are allowed but not recommended.
+	JobName   string `json:"job_name"`
+	ClusterId string `json:"cluster_id"`
```
Comment (vladimirvshivkov) [inline openstack/mrs/v1/job/Create.go:75]: why here ClusterId?
but on the line 27 ClusterID?
and everywhere around the PR
Id or ID?

Comment (Aloento) [reply openstack/mrs/v1/job/Create.go:75]: Id

**Comment by anton-sidelnikov** [comment]
You refactored too many files not related to MRS, please exclude it from PR

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Review by artem-lifshits — APPROVED** [review]

**Review by bakhterets — COMMENTED** [review]
viewed

**Pull request merged**